File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    if( x < 0)
    return false;

    let res = x;
    let res1 = 0;

    while(x > 0){
        let digit = x % 10;
        res1 = res1 * 10 + digit;
        x = Math.floor(x / 10);
    }
    return res === res1;
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        
        temp = x
        res = 0
        while (temp > 0) :  
            
            ld = temp % 10
            res = res * 10 + ld
            temp = temp // 10

            
        if x == res:
            return True

        else:
            return False